tensorflow/compiler/tf2tensorrt/convert/convert_nodes.cc

```diff
@@ -6036,27 +6037,34 @@ Status ConvertResize(OpConverterParams* params) {
                                  node_def.name());
   }
 
+  // Validate inputs_tensor
```

Comment:
Can do you these:
  1. Add a period to the comment to be consistent.
  2. Add a TODO for supporting dynamic shape here.
  3. Modify the PR description, to reflect the change to the converter. It currently only says that the PR adds test cases.

Reply:
1. Fixed
2. Fixed
3. Fixed